orchagent/main.cpp

```diff
@@ -99,6 +111,7 @@ void sighup_handler(int signo)
      */
     gLogRotate = true;
     gSaiRedisLogRotate = true;
```

Comment:
Not for this PR, but these should probably be of type `volatile sig_atomic_t`

https://en.cppreference.com/w/cpp/utility/program/signal

Reply:
https://github.com/Azure/sonic-swss/issues/2045